config/database.yml

```diff
@@ -0,0 +1,33 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: 8
+  url: <%= ENV.fetch('DATABASE_URL', 'postgresql://talk:talk@localhost') %>
+
+development:
+  <<: *default
+  database: talk_development
+
+panoptes_development:
+  <<: *default
+  database: talk_panoptes_development
+
+test:
+  <<: *default
+  database: talk_test
+
+panoptes_test:
+  <<: *default
+  database: talk_panoptes_test
+
+staging:
+  <<: *default
+
+panoptes_staging:
+  <<: *default
+
+production:
+  <<: *default
+
+panoptes_production:
+  <<: *default
```

Comment:
All the `panoptes_*(env)` config settings here need to point to the relevant panoptes db (sad but true), not the talk db config url. I believe there is a custom ENV var `PANOPTES_DATABASE_URL` for this?

Reply:
✔️